src/components/dapps/ModalCreateDapps.vue

```diff
@@ -388,12 +239,15 @@ export default defineComponent({
 
     const openOption = ref(false);
 
-    const unit_d = 3;
-    const decimal = 12;
+    const unitToken = inject('unitToken', '');
```

Comment:
I'm not sure if it's good to use the `inject` method for the same data in different components https://github.com/PlasmNetwork/plasm-apps-vue/blob/1209def72e639ffa03174ae21dbd52771e39cebb/src/components/balance/ModalTransferAmount.vue#L133
when multiple components are providing the same data as well
https://github.com/PlasmNetwork/plasm-apps-vue/blob/1209def72e639ffa03174ae21dbd52771e39cebb/src/components/balance/BalancePlasm.vue#L120
https://github.com/PlasmNetwork/plasm-apps-vue/blob/c69f6ea0382b7cf6abd2dce862cd360fc6325b94/src/components/dapps/CreateDappsTab.vue#L122
I suggest refactoring the token unit selection as a custom event and create a single event hook for providing chain metadata (maybe something like `useChainMetadata.ts` for reading chain name, decimal value, address prefix, etc.?)

Reply:
refactored